README.md

```diff
@@ -1,67 +1,91 @@
 [![codecov](https://codecov.io/gh/brainglobe/brainrender/graph/badge.svg)](https://codecov.io/gh/brainglobe/brainrender)
-[![License: GPL v3](https://img.shields.io/badge/License-GPLv3-blue.svg)](https://www.gnu.org/licenses/gpl-3.0)
-[![Code style: black](https://img.shields.io/badge/code%20style-black-000000.svg)](https://github.com/python/black)
+[![Downloads](https://static.pepy.tech/badge/brainrender)](https://pepy.tech/project/brainrender)
 
+&nbsp;
+&nbsp;
 
-**`brainrender` is a python package for the visualization of three dimensional neuro-anatomical data. It can be used to render data from publicly available data set (e.g. Allen Brain atlas) as well as user generated experimental data. The goal of brainrender is to facilitate the exploration and dissemination of neuro-anatomical data by providing a user-friendly platform to create high-quality 3D renderings.**
+![Example gallery](https://iiif.elifesciences.org/lax/65751%2Felife-65751-fig3-v3.tif/full/,1500/0/default.jpg)
 
-## Docs
-:books: [brainrender docs](https://brainglobe.info/documentation/brainrender/index.html)
+From: Claudi et al. (2021) Visualizing anatomically registered data with brainrender. eLife
+
+
+## Documentation
+
+BrainRender is a project of the BrainGlobe Initiative, which is a collaborative effort to develop a suite of Python-based software tools for computational neuroanatomy. A comprehensive online documentation for brainrender can be found on the BrainGlobe website [here](https://brainglobe.info/documentation/brainrender/index.html).
+
+Furthermore, an open-access journal article describing BrainRender has been published in eLife, available [here](https://doi.org/10.7554/eLife.65751).
 
-📖 [brainrender paper](https://doi.org/10.7554/eLife.65751)
 
 ## Installation
-You can [install `brainrender`](https://brainglobe.info/documentation/brainrender/installation.html) with:
+
+From PyPI:
 
 ```
 pip install brainrender
 ```
 
-## Contributing
-Contributions to brainrender are more than welcome. Please see the [developers guide](https://brainglobe.info/community/developers/index.html). Note that some tests are only run locally, by specifying `--runslow --runlocal` in `pytest`.
+If you encounter any issues, please consult our troubleshooting guide [here](https://brainglobe.info/documentation/brainrender/installation.html).
+
+
+## Quickstart
+
+``` python
+from brainrender import Scene
+from brainrender.actors import Neuron
+
+# Display the Allen Brain mouse atlas.
+scene = Scene(atlas_name="allen_mouse_25um")
+
+# Highlight the cerebral cortex.
+scene.add_brain_region("CTX", alpha=0.2, color="green")
+
+# Add a neuron morphological reconstruction,
+# with coordinates pre-registered to the Allen Brain Atlas.
+neuron = Neuron("/path/to/neuron/morphology.swc")
+scene.add(neuron)
+
+# Display the figure.
+scene.render()
+
+# Export to PNG.
+scene.screenshot("figure.png")
+
+# Export to HTML.
+scene.screenshot("figure.html")
```

Comment:
I think it might be a good idea to have something that can just be copied and pasted, rather than requiring the user to find a suitable swc file.

I also removed the figure saving because it didn't work properly for me. I will raise an issue, but I didn't want to add something potentially broken to the quickstart. 

```suggestion
``` python
from brainrender import Scene

# Display the Allen Brain mouse atlas.
scene = Scene(atlas_name="allen_mouse_25um")

# Display a brain region
primary_visual = scene.add_brain_region("VISp")

# Add label to the brain region
scene.add_label(primary_visual, "Primary visual cortex")

# Display the figure.
scene.render()
```

Reply:
You make a strong point. However, I also think users need to see potential entry points for their data. They are not on the brainrender github page to make pretty pictures of the Allen Brain Atlas; rather they want to know how to visualize their own data. So give them something. `Points` maybe?

Reply:
>  They are not on the brainrender github page to make pretty pictures of the Allen Brain Atlas

You'd be surprised! 

I agree though. I've updated my suggestion. It's a bit long, but includes a variety of concepts.